conf/tls/tls_config.go

```diff
@@ -0,0 +1,125 @@
+package tls
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"io/ioutil"
+
+	"github.com/squareup/pranadb/errors"
+
+	"github.com/alecthomas/kong"
+)
+
+// TLSConfig can be used to build a TLS configuration.
+type TLSConfig struct {
+	// nolint: golint
+	CACert     NamedFileContent `help:"PEM-encoded CA certificates to use for ${tls_target}." placeholder:"FILE"`
```

Comment:
Is this mandatory?

Reply:
Yes, we [add the CA cert to the root CAs](https://go.dev/play/p/Tk9CR4BUyU). This is so that the clients can validate the server certificate.